src/vdom/remove.js

```diff
@@ -0,0 +1,3 @@
+export default function remove(el) {
+	el.parentNode && el.parentNode.removeChild(el);
```

Comment:
JS abstraction leaks so this should be `if (el.parentNode !== null) { el.parentNode.removeChild(el) }` @developit 😄